templates/dashboard.php

```diff
@@ -35,6 +35,7 @@
     $meta_name  = [];
     $meta_id    = [];
     $meta_key   = [];
+    $post_data  = [];
```

Comment:
There is already a variable inside the loop with nearly the same name `$postdata = get_object_vars( $post );`. Rather it should be like `$template_args` as we are passing it to the template.

Reply:
@sapayth Please fix this issue.

Reply:
Done